spec/models/activity_spec.rb

```diff
@@ -496,11 +496,29 @@
         end
       end
 
+      context "when there is a call and total applications is blank" do
+        subject(:activity) { build(:project_activity, call_present: true, total_applications: nil) }
+        it "should not be valid" do
+          expect(activity.valid?(:total_applications_and_awards_step)).to be_falsey
```

Comment:
`valid?` will only ever return `true` or `false`, so you could us `be false` here.

`truthy` and `falsey` [aren't exactly the same](https://eddyluten.com/rspec-be_truthy-exists-or-be-true) as `true` and `false`

Reply:
I used `be_falsey` to keep this test inline with the rest of the tests in this file. Almost all use the terms `truthy` or `falsey`. Shall I change mine to `be_false`?

Reply:
I'd stick with consistency 👍